specification/connector_sync_job/_types/SyncJob.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Id } from '@_types/common'
+import { long } from '@_types/Numeric'
+import { Dictionary } from '@spec_utils/Dictionary'
+import { UserDefinedValue } from '@spec_utils/UserDefinedValue'
+import {
+  ConnectorConfiguration,
+  FilteringConfig,
+  IngestPipelineParams,
+  SyncStatus
+} from '../../connector/_types/Connector'
+
+interface SyncJobConnectorReference {
+  configuration: ConnectorConfiguration
+  filtering: FilteringConfig
+  id: Id
+  index_name: string
+  language?: string
+  pipeline: IngestPipelineParams
+  service_type: string
+}
+
+enum SyncJobType {
+  full,
+  incremental,
+  access_control
+}
+
+enum TriggerMethod {
+  on_demand,
+  scheduled
+}
+
+export interface ConnectorSyncJob {
+  cancelation_requested_at?: string
```

Comment:
Yup it can be `DateTime`, I adapted Sync Job representation to use this type. 

Note, there are several properties in `Connector` representation that should use it as well. I will follow up in a separate PR since I need to reflect a few changes that were merged in to ES after #2367

Reply:
@jedrazb I did that already a few days ago 🙂 https://github.com/elastic/elasticsearch-specification/pull/2507

Reply:
Ahh thank you! Approved again!